Gemfile.lock

```diff
@@ -79,578 +79,677 @@ GEM
       activerecord (>= 3.0)
     addressable (2.7.0)
       public_suffix (>= 2.0.2, < 5.0)
-    after_commit_everywhere (0.1.5)
+    after_commit_everywhere (1.0.0)
```

Comment:
There are a lot of gem updates in here, are they all related to ES 7? Did you have to explicitly manually update any other gems for version compatibility issues?

Reply:
No, those all came along for the ride when I did `bundle update` - I've now merged with master so the extra changes should be gone now (in particular all the ones for newfangled AWS services!)